### PR TITLE
type(test): Adding Integration Template for future Integration tests

### PR DESCRIPTION
**Pull request by tysonjhayes**
This adds a template for how we expect future integration tests to look.
It allows us to show best practices for backwards compatability and allows
 us to formalize the template.

ISSUE_KEY: https://github.com/PowerShell/xNetworking/issues/45

### CODE REVIEW

**Comment by tysonjhayes** [comment]
@PlagueHO - As you were wanting to use my last integration test as a template this is relavent to your interests.

**Comment by PlagueHO** [comment]
Nice! :+1:

File: Templates/integration_template.ps1 [hunk 1]
```diff
@@ -0,0 +1,97 @@
+# Template for Integration Testing
+
+# Copy to Program Files for WMF 4.0 Compatability as it can only find resources in a few known places.
+$DSCModuleName = 'xNetworking'
+$DSCResourceName = 'x<ResourceName>'
```
Comment (KarolKaczmarek) [inline Templates/integration_template.ps1]: I'd add # TODO comment to the places which need to be updated

Comment (PlagueHO) [inline Templates/integration_template.ps1]: I was just working on these templates as part of Issue #47 and have set a +++ to indicate that. But I'll convert it to TODO as I that is the global "Fix me up" :+1:

**Pull request merged**